src/CMakeLists.txt

```diff
@@ -19,11 +19,18 @@
 # http://numenta.org/licenses/
 # -----------------------------------------------------------------------------
 
-cmake_minimum_required(VERSION 3.3)
+cmake_minimum_required(VERSION 3.7)
 project(nupic_core CXX)
 
+
+message(STATUS "------- nupic_core C++ library -------")
+message(STATUS "CMAKE_SYSTEM             = ${CMAKE_SYSTEM}")
+message(STATUS "CMAKE_BINARY_DIR         = ${CMAKE_BINARY_DIR}")
+message(STATUS "CMAKE_CURRENT_SOURCE_DIR = ${CMAKE_CURRENT_SOURCE_DIR}")
+message(STATUS "EP_BASE                  = ${EP_BASE}")
```

Comment:
what is EP_BASE?

Reply:
Its where external stuff gets built.  build/scripts/ThirdParty